includes/admin/OrderSettings.php

```diff
@@ -326,11 +334,19 @@ public function setBillingRecipient(): self
     {
         $consignment            = ConsignmentFactory::createByCarrierName($this->carrier);
         $localCountryCode       = $consignment->getLocalCountryCode();
-        $this->billingRecipient = (new RecipientFromWCOrder(
-            $this->order,
-            $localCountryCode,
-            RecipientFromWCOrder::BILLING
-        ));
+        try {
+            $this->billingRecipient = (new RecipientFromWCOrder(
+                $this->order, $localCountryCode, RecipientFromWCOrder::BILLING
```

Comment:
Kunnen deze 2 brokken niet gewoon in 1 functie? Zodat je alleen dit hoeft aan te roepen:

```php
$this->shippingRecipient = $this->createRecipient(RecipientFromWCOrder::SHIPPING);
// ...
$this->billingRecipient = $this->createRecipient(RecipientFromWCOrder::BILLING);
```

edit: zie nu dat het al zo was, dus je hoeft het niet meteen helemaal te refactoren, maar als je dat wel ziet zitten... graag 👼🏼